docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -248,6 +252,31 @@ To manage the media controller features in your application, you must learn to s
 
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+<a name="setting_content_type"></a>
+
+## Setting Content Type for the Currently Playing Media
```

Comment:
Just a question. In the list above, this "content type" sections comes after the "setting age rating". But here, the "content type" come before the "setting age rating". In there any reason the order is different?

Reply:
I accidentally put it in the wrong order. I updated the change by switching the order on the list.